css/style.css

```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.margin-top-12 {
+  margin-top: 12px;
+}
+
+.wrap {
+  padding: 6px 24px;
+}
+
+/* Header flex container */
+.header-nav {
+  background-color: #fff;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
```

Comment:
- [ ] [REQUIRED] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.flex` class in multiple places.

- (your code)

```css
.header-nav {
  /* ...code */
  display: flex;
  /* ...code */
}

.headline-wrap {
  /* ...code */
  display: flex;
  /* ...code */
}
```

- suggestion

```css
.flex {
  display: flex;
}
```